### ISSUE TITLE
Error with OpenProject 4.1

### ISSUE THREAD

**Issue by JohnDoe42** [oliverguenther/openproject-revisions_git#25]
```
Started GET "/projects/testproject/repository" for 172.16.0.1 at 2015-05-21 21:00:23 +0200
Started GET "/" for 127.0.0.1 at 2015-05-21 21:00:23 +0200
Processing by WelcomeController#index as HTML
  Rendered welcome/index.html.erb within layouts/base (124.6ms)
  Rendered search/_mini_form.html.erb (1.2ms)
  Rendered hooks/login/_auth_provider.html.erb (0.4ms)
  Rendered account/_auth_providers.html.erb (4.1ms)
  Rendered account/_login.html.erb (15.9ms)
  Rendered layouts/_action_menu.html.erb (0.4ms)
Completed 200 OK in 283.9ms (Views: 186.6ms | ActiveRecord: 25.3ms)
Processing by RepositoriesController#show as HTML
  Parameters: {"project_id"=>"testproject"}
  Rendered repositories/_breadcrumbs.html.erb (15.8ms)
  Rendered repositories/_navigation.html.erb (21.8ms)
  Rendered vendor/bundle/ruby/2.1.0/bundler/gems/openproject-revisions_git-d06af74e6ba0/app/views/repositories/show.html.erb within layouts/base (34.5ms)
Completed 500 Internal Server Error in 467.7ms

ActionView::Template::Error (tried to call method kind, but was not found!):
    30: <%= link_to 'root', :action => 'show', :project_id => @project, :path => nil, :rev => @rev %>
    31: <%
    32: dirs = path.split('/')
    33: if 'file' == kind
    34:     filename = dirs.pop
    35: end
    36: link_path = ''
  app/views/repositories/_breadcrumbs.html.erb:33:in `_app_views_repositories__breadcrumbs_html_erb__2130571149946120337_71560080'
  app/views/repositories/_navigation.html.erb:34:in `_app_views_repositories__navigation_html_erb___2656409987385933882_71848680'
  app/middleware/params_parser_with_exclusion.rb:40:in `call'
```

Just have updated to stable/4.1

**Comment by JohnDoe42** [comment]
To fix this, change line 34 in the openproject file app/views/repositories/_navigation.html.erb from

```
<% crumbs = render :partial => 'breadcrumbs', :locals => { :path => @path, :revision => @rev }.merge(merge) %>
```

to

```
<% crumbs = render :partial => 'breadcrumbs', :locals => { :path => @path, :kind => 'dir',:revision => @rev }.merge(merge) %>
```

**Comment by oliverguenther** [comment]
Thanks for spotting, this is best solved in the `show.html.erb`, should be fixed in 07b9946
